apps/vscode-extension/src/commands/helpers.ts

```diff
@@ -29,7 +29,50 @@ export const wrapCommandWhenExplorerContext = (
 /**
  * Get the runner for NPM scripts
  */
-export const getPackageScriptRunner = () => {
-  const packageManager = vscode.workspace.getConfiguration('otter').get<string>('packageManager', 'npm');
+export const getPackageScriptRunner = async (): Promise<string> => {
+  const [workspaceJsonUri] = await vscode.workspace.findFiles('{angular,nx}.json');
```

Comment:
we should make sure nx has priority if both are present. you can have angular.json file present for edge cases like workaround for working with ng update, templates, mock files etc

Reply:
I would have say the opposite 
first `angular.json` then `nx.json`
as we said we should support first Angular and then Nx if possible

Reply:
I have created a method to sort everytime the same way 
let's discuss the order all together

Reply:
As discussed we keep `angular.json` first for the moment but it could change later if we face some issue about it